### PR TITLE
Deploying new neurodamus 3 series

### PR DESCRIPTION
**Pull request by ferdonline**

### CODE REVIEW

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -176,3 +178,8 @@ packages:
       - neurodamus-neocortex+coreneuron@0.3^coreneuron+knl
       - neurodamus-hippocampus+coreneuron@0.4^coreneuron+knl
       - neurodamus-thalamus+coreneuron@0.3^coreneuron+knl
+      - neurodamus-core~common@3.0.0
+      - neurodamus-neocortex+coreneuron@1.0
+      - neurodamus-hippocampus+coreneuron@1.0
+      - neurodamus-thalamus+coreneuron@1.0
+      - neurodamus-mousify+coreneuron@1.0
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml:185]: Are these ever deployed without `+coreneuron`? May be worth changing the default in the future (does not have to be this PR)

**Review by matz-e — APPROVED** [review]

File: deploy/packages/bbp-packages.yaml (hunk 1 above)
Comment (ferdonline) [reply deploy/packages/bbp-packages.yaml:185]: I think deployed not... but when building locally, personally I find that the default ~Coreneuron makes more sense

**Comment by pramodk** [comment]
we do want to get newer neuron and coreneuron deployed. Is that included now?

File: deploy/packages/bbp-packages.yaml (hunk 1 above)
Comment (matz-e) [reply deploy/packages/bbp-packages.yaml:185]: Fair enough! The new deployment will use environments (when I finally pull the trigger, waiting for some upstream goodies), and we should be able to fix it there.

Within environments, we can actually override parts of `packages.yaml` and pin `+coreneuron` there — that will not affect users, and make these specs easier to read :)

**Pull request merged**